crates/ryot/Cargo.toml

```diff
@@ -19,8 +19,7 @@ serde_repr = "0.1"
 ryot_assets = { path = "../ryot_assets", version = "0.2.0" }
```

Comment:
is `version =` necessary when doing paths?

Reply:
It's not needed, but it doesn't hurt keeping it, it also helps guaranteeing we are always release the same version to all of them.